### PR TITLE
feat: process unchecked block with cache

### LINKED ISSUE #1176: process unchecked block with cache
when sync block,  unchecked block store to badger immediately may cause performance issue, so use cache to process unchecked block

### PR DESCRIPTION
**Pull request by zengchen221**
### Proposed changes in this pull request

- process unchecked block with cache, fix #1176 
- remove badger v1.6 library

### Type
- [ ] Bug fix: (Link to the issue #{issue No.})
- [x] Feature (Non-breaking change)
- [ ] Feature (Breaking change)
- [ ] Documentation Improvement

### Checklist
Write a small comment explaining if its `N/A` (not applicable)

- [x] Read the [CONTRIBUTION](https://github.com/qlcchain/go-qlc/blob/master/CONTRIBUTING.md).
- [ ] All the tests are passing after the introduction of new changes.
- [ ] Added tests respective to the part of code I have written.
- [ ] Added proper documentation where ever applicable.
- [x] Code has been written according to [Golang-Style-Guide](https://github.com/qlcchain/go-qlc/blob/master/CONTRIBUTING.md#code-standard)

### Extra information
Any extra information related to this pull request.

### CODE REVIEW

**Review by gythialy — APPROVED** [review]
TLDR;
